cmd/activator/main.go

```diff
@@ -244,7 +243,7 @@ func main() {
 	if err != nil {
 		logger.Fatalw("Unable to create request log handler", zap.Error(err))
 	}
-	ah = reqLogHandler
+	ah = activatorhandler.NewRequestMetricHandler(revisionInformer.Lister(), reporter, logger, reqLogHandler)
```

Comment:
well, first, it should be the outermost handler.

Reply:
see expected behaviour https://github.com/knative/serving/issues/4837

Reply:
How about making this the outermost handler and filtering probe and health check requests?